layouts/index.html

```diff
@@ -1,633 +1,257 @@
-            <a class="link" href="/language/python/">Containerize a Python app using Docker</a>
-          </li>
-          <li>
-            <a class="link" href="/language/java/">Containerize a Java app using Docker</a>
-          </li>
-          <li>
-            <a class="link" href="/language/golang/">Containerize a Go app using Docker</a>
-          </li>
-          <li>
-            <a class="link" href="/samples/">View more languages and frameworks in Docker
-              samples</a>
-          </li>
-        </ul>
-      </div>
-    </div>
-    <div class="flex mx-auto w-[1200px] flex-col p-6 lg:w-full">
-      <h2 class="text-3xl">Play with Docker</h2>
-      <div class="grid grid-cols-3 gap-4 p-6 sm:flex sm:flex-col xl:grid-cols-2">
-        <div>
-          <a class="h-full" href="/get-started/">
-            <div
-              class="flex h-full flex-col gap-2 rounded border border-gray-light-100 bg-white p-4 drop-shadow-sm hover:border-gray-light-200 hover:drop-shadow-lg dark:border-gray-dark-400 dark:bg-gray-dark-200 hover:dark:border-gray-dark">
-              <div class="flex items-center gap-4">
-                <div>
-                  <div class="text-xl leading-snug text-gray-light-800 dark:text-white">
-                    Tutorial
+            <a class="h-full" href="/desktop/">
+              <div
+                class="flex h-full flex-col gap-2 rounded border border-gray-light-100 bg-white p-4 drop-shadow-sm hover:border-gray-light-200 hover:drop-shadow-lg dark:border-gray-dark-400 dark:bg-gray-dark-200 hover:dark:border-gray-dark">
+                <div class="flex items-center gap-4">
+                  <span class="icon text-[64px] text-violet-light dark:text-violet-dark" data-icon="description"></span>
+                  <div>
+                    <div class="text-xl leading-snug text-gray-light-800 dark:text-white">
+                      Manuals
+                    </div>
                   </div>
                 </div>
-              </div>
-              <div class="leading-snug text-gray-light-500 dark:text-gray-dark-700">
-                Self-paced tutorials to increase your Docker knowledge.
-              </div>
-            </div>
-          </a>
-        </div>
-        <div>
-          <a class="h-full" href="https://labs.play-with-docker.com/">
-            <div
-              class="flex h-full flex-col gap-2 rounded border border-gray-light-100 bg-white p-4 drop-shadow-sm hover:border-gray-light-200 hover:drop-shadow-lg dark:border-gray-dark-400 dark:bg-gray-dark-200 hover:dark:border-gray-dark">
-              <div class="flex items-center gap-4">
-                <div>
-                  <div class="text-xl leading-snug text-gray-light-800 dark:text-white">
-                    Lab environment
-                  </div>
+                <div class="leading-snug text-gray-light-500 dark:text-gray-dark-700">
+                  Browse the manuals and learn how to use Docker products.
```

Comment:
suggestion: the description of guides and manuals doesn't help me differentiate them enough. do you think this could be more specific or even list an example, like `...learn how to use Docker products, like Docker Desktop, Engine, and Scout.`? or maybe even `Browse the manuals of each Docker product and learn how to use them`?

(for context: as a docker dev, i _still_ find myself confused about what a guide is and what a manual is whenever i go to the docs 😅 or, put another way, when i'm looking for a specific answer, i don't understand the taxonomy of information enough to pick the right place to look. the landing pages for each of those sections do make the taxonomy much clearer! so i wonder if tweaking these landing page entrypoints may make a difference.)

i know this is a hard problem to solve in general! so feel free to take or leave my suggestion.